Training/Training/file.c

```diff
@@ -3,39 +3,29 @@
 // Copyright (c) Gokul TS @Metamation India.
 // ------------------------------------------------------------------
 // file.c
-// C Program to implement Bubble sort and binary search algorithm with test cases
+// C Program to check test cases
 // ------------------------------------------------------------------------------------------------
+#include "SortAndSearch.h"
 #include <stdio.h>
 #include <stdint.h>
 #include <string.h>
 #include <ctype.h>
 #include <stdlib.h>
 
-#define MAX_DIGIT 13
-#define TEST_SIZE 7
-#define ANSI_RESET_ALL "\x1b[0m"
-#define ANSI_COLOR_RED "\x1b[31m"
-#define ANSI_COLOR_GREEN "\x1b[32m"
-#define ANSI_COLOR_YELLOW "\x1b[33m"
-
-/// <summary>Changes the input array into a sorted array</summary>
-void BubbleSort (int arr[], int size);
-/// <summary>Returns the index of key element in an array</summary>
-int BinarySearch (int sortArr[], int key, int size);
-/// <summary>Gets integer input from user</summary>
-int GetUserInput (char* prompt);
+/// <summary>Gets integer input from user with specified prompt</summary>
+int GetUserInputWithPrompt (char* prompt);
 /// <summary>Implements the search and sort algorithm</summary>
 void SearchAndSort ();
 /// <summary>Clears the content on screen</summary>
 void ClrScreen ();
 /// <summary>Checks test cases</summary>
 void TestCases ();
-/// <summary>Compares two arrays and returns the result</summary>
+/// <summary>Returns whether two arrays are equal or not</summary>
 int CompareArrays (int* input, int* output, int size);
 
 int main () {
    while (1) {
-      int num = GetUserInput ("1-SORT AND SEARCH\n2-TEST CASES\n3-CLEAR SCREEN\n4-EXIT\nEnter an option:");
+      int num = GetUserInputWithPrompt ("1-SORT AND SEARCH\n2-TEST CASES\n3-CLEAR SCREEN\n4-EXIT\nEnter an option:");
       if (num > 4 || num < 1)
          printf (ANSI_COLOR_RED"Please enter a number between 1-4\n"ANSI_RESET_ALL);
       else {
```

Comment:
The else statement here is a single line statement as it only has 1 switch statement inside it. So brackets can be removed.